src/main/java/org/jitsi/videobridge/OctoChannel.java

```diff
@@ -0,0 +1,317 @@
+        = new OctoDatagramPacketFilter(true);
+
+    /**
+     * The {@link Logger} to be used by this instance to print debug
+     * information.
+     */
+    private final Logger logger;
+
+    /**
+     * Initializes a new <tt>Channel</tt> instance which is to have a specific
+     * ID. The initialization is to be considered requested by a specific
+     * <tt>Content</tt>.
+     *
+     * @param content the <tt>Content</tt> which is initializing the new
+     * instance
+     * @param id the ID of the new instance. It is expected to be unique
+     * within the
+     * list of <tt>Channel</tt>s listed in <tt>content</tt> while the new
+     * instance
+     * is listed there as well.
+     * @throws Exception if an error occurs while initializing the new instance
+     */
+    public OctoChannel(Content content, String id)
+        throws Exception
+    {
+        super(
+                content, id, null /*channelBundleId*/,
+                OctoTransportManager.NAMESPACE, false /*initiator*/);
+
+        conferenceId = content.getConference().getGid();
+        mediaType = content.getMediaType();
+        logger
+            = Logger.getLogger(classLogger, content.getConference().getLogger());
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public void describe(ColibriConferenceIQ.ChannelCommon commonIq)
+    {
+        super.describe(commonIq);
+
+        // TODO: uncomment after the mvn versions are updated
+        //commonIq.setType(ColibriConferenceIQ.OctoChannel.TYPE);
+    }
+
+    /**
+     * @return The Octo ID of the conference.
+     */
+    public String getConferenceId()
+    {
+        return conferenceId;
+    }
+
+    /**
+     * Sets the list of remote Octo relays that this channel should transmit to.
+     * @param relayIds the list of strings which identify a remote relay.
+     */
+    public void setRelayIds(List<String> relayIds)
+    {
+        OctoTransportManager transportManager = getOctoTransportManager();
+
+        transportManager.setRelayIds(relayIds);
+    }
+
+    /**
+     * @return The transport manager of this {@link OctoChannel} as a
+     * {@link OctoTransportManager} instance.
+     */
+    private OctoTransportManager getOctoTransportManager()
+    {
+        if (transportManager == null)
+        {
+            transportManager = (OctoTransportManager) getTransportManager();
+        }
+        return transportManager;
+    }
+
+    /**
+     * @return the {@link MediaType} of this channel.
+     */
+    public MediaType getMediaType()
+    {
+        return mediaType;
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public RtpChannelDatagramFilter getDatagramFilter(boolean rtcp)
+    {
+        return rtcp ? rtcpFilter : rtpFilter;
+    }
+
+    /**
+     * TODO: this should be in a utility somewhere.
+     * @param data
+     * @param off
+     * @param len
+     * @return
+     */
+    private boolean isRTCP(byte[] data, int off, int len)
```

Comment:
agreed, i actually thought we had one already.  what about putting it in `RTPUtils`?

Reply:
Yes! Will fix